vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -146,9 +155,28 @@ IAsyncOperation<HttpRequestMessage> WinRTHttpResource::CreateRequest(
       auto file = co_await StorageFile::GetFileFromApplicationUriAsync(Uri{to_hstring(data["uri"].asString())});
       auto stream = co_await file.OpenReadAsync();
       content = HttpStreamContent{std::move(stream)};
-    } else if (!data["form"].empty()) {
-      // #9535 - HTTP form data support
-      // winrt::Windows::Web::Http::HttpMultipartFormDataContent()
+    } else if (!data["formData"].empty()) {
+      winrt::Windows::Web::Http::HttpMultipartFormDataContent multiPartContent;
+      auto formData = data["formData"];
+
+      for (auto &formDataPart : formData) {
```

Comment:
>auto & [](http://example.com/codeflow?start=11&length=6)

Can it be 'auto const&'?

Reply:
Turns out no.

Making it `const` would throw an exception for non-existing object keys.
Not sure why, but will keep as `auto&`, same as `data`.